README.md

```diff
@@ -17,8 +17,41 @@ Table of contents:
 
 ## Usage
 
+Since this is a tool for helping the developer of a library or application
+write better code, it is recommended to include clippy as an optional
+dependency.
```

Comment:
Well, that — here — seems to imply you still need the dependency when using `cargo clippy`.